templates/contactDetails.html

```diff
@@ -1,35 +1,33 @@
-<div ng-if="ctrl.contact===undefined && !ctrl.loading">
-	<div id="emptycontent" class="">
-		<div class="icon-contacts-dark"></div>
-		<h2>{{ctrl.t.noContacts}}</h2>
+<div class="contact-details-wrapper wrapper-show" ng-show="ctrl.show">
+	<div ng-if="ctrl.contact===undefined && !ctrl.loading">
+		<div id="emptycontent" class="">
+			<div class="icon-contacts-dark"></div>
+			<h2>{{ctrl.t.noContacts}}</h2>
+		</div>
 	</div>
-</div>
-<div ng-if="ctrl.contact!==undefined">
-<header class="contactdetails__header" ng-style="{'background-color': (ctrl.contact.uid() | contactColor)}">
-	<img ng-if="ctrl.photo!==undefined" class="contactdetails__logo avatar" data-ng-src="data:image/png;base64,{{ctrl.photo}}" />
-	<h2>
-		<input type="text" id="details-fullName" class="contactdetails__name" placeholder="{{ctrl.t.placeholderName}}" autocomplete="off"
-		name="email" ng-model="ctrl.contact.fullName" ng-model-options="{ getterSetter: true, debounce: 500 }" ng-change="ctrl.updateContact()" value="" />
-	</h2>
-	<div>
-		<input type="text" id="details-org" class="contactdetails__org" placeholder="{{ctrl.t.placeholderOrg}}" autocomplete="off"
-			   name="email" ng-model="ctrl.contact.org" ng-model-options="{ getterSetter: true, debounce: 500 }" ng-change="ctrl.updateContact()" value="" />
-		<input type="text" id="details-title" class="contactdetails__title" placeholder="{{ctrl.t.placeholderTitle}}" autocomplete="off"
-			   name="email" ng-model="ctrl.contact.title" ng-model-options="{ getterSetter: true, debounce: 500 }" ng-change="ctrl.updateContact()" value="" />
-	</div>
-	<button ng-click="ctrl.deleteContact()" class="icon-delete-white" title="Delete"></button>
-</header>
-<section>
-	<div ng-repeat="prop in ctrl.contact.props | toArray | orderDetailItems:'$key'">
-		<detailsItem ng-repeat="propData in prop" name="prop.$key" data="propData" model="ctrl" index="$index" ng-class="[ 'details-item-' + prop.$key ]"></detailsItem>
-	</div>
-	<div class="select-addressbook" ng-if="ctrl.addressBooks.length > 1">
-		<select ng-model="ctrl.addressBook" ng-change="ctrl.changeAddressBook(ctrl.addressBook)" ng-options="book.displayName for book in ctrl.addressBooks">
+	<div ng-if="ctrl.contact!==undefined">
+	<header class="contactdetails__header" ng-style="{'background-color': (ctrl.contact.uid() | contactColor)}">
+		<button class="details-back" ng-click="ctrl.clearContact()">Back</button>
```

Comment:
Can I suggest using this instead?
`<div id="app-navigation-toggle-back" class="details-back icon-download" ng-click="ctrl.clearContact()"></div>`
```
#app-navigation-toggle-back {
    position: fixed;
    display: inline-block !important;
    top: 90px;
    left: 0;
    width: 44px;
    height: 44px;
    z-index: 149;
    background-color: rgba(255, 255, 255, .7);
    cursor: pointer;
    opacity: .6;
    transform: rotate(90deg);
}
```
![contacts owncloud](https://cloud.githubusercontent.com/assets/14975046/14103628/52b626fc-f5a1-11e5-88c1-fc8583af71d1.png)